tests/schema/tests.py

```diff
@@ -273,6 +278,16 @@ def get_column_collation(self, table, column):
                 if f.name == column
             )
 
+    def get_column_comment(self, table, column):
+        with connection.cursor() as cursor:
+            columns = connection.introspection.get_table_description(cursor, table)
+        return [c.comment for c in columns if c.name == column][0]
+
+    def assert_column_comment_not_exists(self, table, column):
+        with connection.cursor() as cursor:
+            columns = connection.introspection.get_table_description(cursor, table)
+        self.assertFalse(any([c.name == column and c.comment for c in columns]))
+
```

Comment:
Perhaps this could be aligned with the similar implementation of these in `MigrationTestBase`? Perhaps extract into a mixin that both could use? Lots of `assertEqual()` here could be replaced with `assertColumnCommentEquals()`?